src/closures/constant_diffusivity_closures.jl

```diff
@@ -0,0 +1,83 @@
+#
+# Constant diffusivity
+#
+
+"""
+    ConstantIsotropicDiffusivity(T=Float64; ν=1e-6, κ=1e-7)
+
+Return a `ConstantIsotropicDiffusivity` closure object of type `T` with
+viscosity `ν` and scalar diffusivity `κ`.
+"""
+Base.@kwdef struct ConstantIsotropicDiffusivity{T} <: ScalarDiffusivity{T}
+    ν :: T = 1e-6
+    κ :: T = 1e-7
+end
+
+ConstantIsotropicDiffusivity(T; kwargs...) =
+    typed_keyword_constructor(T, ConstantIsotropicDiffusivity; kwargs...)
+
+κ_ccc(i, j, k, grid, closure::ConstantIsotropicDiffusivity, args...) = closure.κ
+
+∂ⱼ_2ν_Σ₁ⱼ(i, j, k, grid, closure::ConstantIsotropicDiffusivity, u, v, w, T, S) = (
```

Comment:
These functions don't actually multiply ν by 2 but the function name still has `2ν` in it?